src/gui/optionsdialog.cpp

```diff
@@ -1407,9 +1411,14 @@ bool OptionsDialog::WinStartup() const
 }
 #endif
 
-bool OptionsDialog::preventFromSuspend() const
+bool OptionsDialog::preventFromSuspendWhenDownloading() const
```

Comment:
These two methods are redundant. Please remove them.

Reply:
How are they redundant?

Reply:
They are never used and never supposed to be used.

Many of these getters in this class (even if they are still used) are simply remnants of the past...